dynatrace/rest/hybrid_client_test.go

```diff
@@ -141,6 +147,24 @@ func TestHybridClient(t *testing.T) {
 	}
 }
 
+func TestHybridClientActiveGateRequest(t *testing.T) {
```

Comment:
Would be nice to also add some test that fail (classic url == 0, and url.Parse fail). 

I would also add some headers to the test.

Reply:
done 👍